builtin/logical/pki/ca_util.go

```diff
@@ -180,15 +187,15 @@ func getKeyTypeAndBitsForRole(ctx context.Context, b *backend, storage logical.S
 			return "", 0, errors.New("unable to determine managed key id" + err.Error())
 		}
 
-		pubKeyManagedKey, err := getManagedKeyPublicKey(ctx, b, keyId)
+		pubKeyManagedKey, err := getManagedKeyPublicKey(sc.Context, sc.Backend, keyId)
```

Comment:
It would make a lot of sense to me if getManagedKeyPublicKey was a function off storageContext (commenting here since managed_key_util isn't changed) though this is a oss change so, maybe not?